### PR TITLE
[T2] Update skip conditon for QosSaiLossyQueue and QosSaiLossyQueueVoq

### PR DESCRIPTION
**Pull request by zhixzhu**
### Description of PR

Double commit https://github.com/sonic-net/sonic-mgmt/pull/10300 and https://github.com/sonic-net/sonic-mgmt/pull/10487

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305

### CODE REVIEW

**Comment by XuChen-MSFT** [comment]
@zhixzhu 
after apply your changes to latest 202305 branch, and ran qos sai test, Encounter testQosSaiLossyQueueVoq error on Cisco-8102-C64 device,  and there are some conflicts, can you fix them?
```
==================================== ERRORS ====================================
_ ERROR at setup of TestQosSai.testQosSaiLossyQueueVoq[single_asic-lossy_queue_voq_1] _

duthost = <MultiAsicSonicHost bjw-can-8102-1>
mg_facts = {'deployment_id': '1', 'dhcp_servers': [], 'dhcpv6_servers': [], 'forced_mgmt_routes': ['10.150.22.0/23', '10.50.50.50', '10.50.10.50'], ...}

    @pytest.fixture(scope="function")
    def singleMemberPort(duthost, mg_facts):
        '''
            Installs static route for a port that is either a single-member lag or not part of a port channel.
        '''
        dst_port = None
        # Try to find a port with no port channel
        all_lag_members = []
        for lag_dict in mg_facts["minigraph_portchannels"].values():
            all_lag_members += lag_dict["members"]
        all_ports = mg_facts["minigraph_ports"].keys()
        non_lag_ports = set(all_ports) - set(all_lag_members)
>       assert len(
            non_lag_ports) > 0, "Failed to find either a single-member lag or a non-lag port"
E       AssertionError: Failed to find either a single-member lag or a non-lag port

all_lag_members = ['Ethernet136', 'Ethernet0', 'Ethernet4', 'Ethernet144', 'Ethernet148', 'Ethernet16', ...]
all_ports  = dict_keys(['Ethernet136', 'Ethernet0', 'Ethernet4', 'Ethernet144', 'Ethernet148', 'Ethernet16', 'Ethernet20', 'Etherne...Ethernet212', 'Ethernet216', 'Ethernet220', 'Ethernet232', 'Ethernet240', 'Ethernet244', 'Ethernet248', 'Ethernet252'])
dst_port   = None
duthost    = <MultiAsicSonicHost bjw-can-8102-1>
lag_dict   = {'members': ['Ethernet252'], 'name': 'PortChannel1024', 'namespace': ''}
mg_facts   = {'deployment_id': '1', 'dhcp_servers': [], 'dhcpv6_servers': [], 'forced_mgmt_routes': ['10.150.22.0/23', '10.50.50.50', '10.50.10.50'], ...}
non_lag_ports = set()
```

**Comment by zhixzhu** [comment]
@XuChen-MSFT conflicts resolved. 
This error is from fixture singleMemberPort, latest 202305 branch doesn't include this fixture in the test case, will not have this error.

**Comment by wangxin** [comment]
@zhixzhu There are conflicts blocking the merge of this PR. Can you help resolve?

**Review by abdosi — DISMISSED** [review]

**Comment by abdosi** [comment]
please check again.

**Comment by zhixzhu** [comment]
@wangxin conflicts resolved.

**Comment by abdosi** [comment]
@zhixzhu please help resolve the conflict

**Comment by zhixzhu** [comment]
@abdosi conflicts resolved.

**Comment by wangxin** [comment]
The other PR I merged today caused new conflicts again. @zhixzhu Can you help resolve the conflicts?

**Comment by zhixzhu** [comment]
@wangxin conflicts resolved.

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -2153,6 +2170,14 @@ def _skip_watermark_multi_DUT(
         yield
         return
 
+    @pytest.fixture(scope="function", autouse=False)
+    def skip_src_dst_different_asic(self, dutConfig):
+        if dutConfig['dutAsic'] != dutConfig['dstDutAsic']:
```
[the remark this answers is not in the document]
Comment (zhixzhu) [reply tests/qos/qos_sai_base.py:2175]: Key 'dstDutAsic' is added to dutConfig, it should always has that key.

**Comment by XuChen-MSFT** [comment]
@abdosi  don't see test failure now

**Comment by abdosi** [comment]
@wangxin please help merge of this.

**Review by abdosi — APPROVED** [review]

**Pull request merged**